docs/reference/api/drivers/UARTSerial.md

```diff
@@ -0,0 +1,54 @@
+## UARTSerial
+
+The UARTSerial class provides an alternative and enhanced version of the [Serial](https://os.mbed.com/docs/v5.6/reference/serial.html) interface. 
+
+
+Features:
+
+* It uses the latest FileHandle API that permits non-blocking and device operations. It is now possible to use a serial device with the C library file handling functions.
+E.g
+
+        UARTSerial serial(RX, TX);
```

Comment:
Lets keep this as a serial example, not mixing the filehandle in

Reply:
AFAIU from Kevin, being able to use filehandles and thus directly call things like fprintf() to write out to the serial port was one of the main advantages of UARTSerial over Serial. If we just make this page look like Serial, why would anyone use it ?